src/sandstorm/run-bundle.c++

```diff
@@ -1183,18 +1189,17 @@ private:
     KJ_SYSCALL(sigprocmask(SIG_SETMASK, &sigset, nullptr));
   }
 
-  void copyEtc() {
+  void linkEtc() {
     auto files = splitLines(readAll("etc.list"));
```

Comment:
Oh, you're right.  I'd say to add `/etc/resolvconf` to the whitelist, and then path traversal will take care of the rest.

Also we only create each symlink if its target exist at startup time, which I think is reasonable, but maybe we want to make dangling symlinks just in case the paths are added later?